README.md

```diff
@@ -1,5 +1,306 @@
+
+The impetus of this application was to create an easy way to spin up Media Services resources on Azure before a broadcast and spin them down again after the show ends. This helps minimize unnecessary billing on our Azure account and puts the power of service management into the hands of the broadcaster. To reduce the costs associated with streaming on Azure, note that ALL resources created for a live streaming session will be DESTROYED when the application stops them. That includes the [video buffer stored in the archive window](#LIVE_STREAMING_API_ARCHIVE_WINDOW_LENGTH). If you want to retain a copy of your live streaming session, it is recommended that the broadcaster records it locally before shutting down the stream and stopping the media resources.
+
+## Basic Definitions
+
+This project controls two major resource types on Azure Media Services: Streaming Endpoints and Live Events. Here are their definitions:
+
+- **Live Events:** A channel that a broadcaster will reuse to publish similar events over and over again.
+- **Streaming Endpoint:** The ingest mechanism on Azure Media Services which allows a broadcaster to connect their streaming software of choice, such as [Wirecast](https://www.telestream.net/wirecast/) or [OBS Studio](https://obsproject.com/), and begin sending a live stream.
+
+## Installation
+
+As mentioned before, this application is intended to run on Azure Functions. Therefore, the instructions are geared toward the process of installing the API there. Overall, there are several significant phases to the installation:
+
+1. [Setting up the Media Services resources on Azure](#Setting-up-the-Media-Services-Resources-on-Azure)
+1. [Creating the Azure Functions application](#Creating-the-Azure-Functions-Application)
+1. [Setting up the environment variables](#Setting-up-the-Environment-Variables)
+1. [Deploying the application](#Deploying-the-Application)
+1. [Authenticating with the application](#Authenticating-with-the-Application)
+1. [Set up the streaming software](#Set-up-the-Streaming-Software)
+
+Guides for using the API as a [broadcaster](https://github.com/literal-life-church/live-streaming-controller) and [viewer](https://github.com/literal-life-church/stream-switch) are available in the READMEs of their respective repositories.
+
+### Setting up the Media Services Resources on Azure
+
+Obviously, to use this application, you'll need an active Azure account. Please [sign up for an account](https://signup.azure.com/) if you don't already have one. Once you have an account set up, [sign in to the Azure Portal](https://portal.azure.com/).
+
+#### Create the Media Service
+
+First, begin by creating the Media Service resource.
+
+1. From the search bar at the top, search for "Media Services," and then select the "Media Services" service
+1. Click the Add button to create a new Media Service
+1. Enter an Account Name, choose your active Subscription, and a Location
```

Comment:
Codacy found an issue: [[ordered-list-marker-value] Marker should be `3`, was `1`](https://app.codacy.com/gh/literal-life-church/live-streaming-api/pullRequest?prid=5728997)